intents.yaml

```diff
@@ -180,6 +180,16 @@ HassSetPosition:
       - "domain"
       - "device_class"
 
+HassGetDate:
+  supported: true
+  domain: homeassistant
+  description: "Gets the current date"
+
+HassGetTime:
+  supported: true
+  domain: homeassistant
+  description: "Gets the current time"
```

Comment:
Considering Home Assistant also has date & time entities, these names seem very unfortunate for the future.

Wouldn't it be better to use `HassGetCurrentDate` and `HassGetCurrentTime` instead?

Reply:
That's a good point. I'll change them :+1:

Reply:
Updated